docs/source/AdministratorGuide/Systems/MonitoringSystem/index.rst

```diff
@@ -84,13 +79,28 @@ can be defined with::
 
 The given periods above are also the default periods in the code.
 
-
-Enable WMSHistory monitoring
+Enable the Monitoring System
 ============================
 
-You have to add ``Monitoring`` to the ``Backends`` option of WorkloadManagement/StatesAccountingAgent.
-If you do so, this agent will collect information using the JobDB and send it to the Elasticsearch database.
-This same agent can also report to the MySQL backend of the Accounting system (which is in fact the default).
+In order to enable the monitoring of all the following types with an ElasticSearch-based backend, you should add the value `Monitoring` to the flag
+`MonitoringBackends` in Operations/Default where the default values is `Accounting`.
+
+This can be done either via the CS or directly in the web app in the Configuration Manager as following::
+
+   Operations
+   {
+     Defaults
+     {
+       MonitoringBackends = Accounting, Monitoring
+     }
+   }
+
+
+WMSHistory & PilotsHistory Monitoring
+=====================================
+
+When enabled, the WorkloadManagement/StatesAccountingAgent will collect information using the JobDB and the PilotAgentsDB and send it to the Elasticsearch database.
+This same agent can also report the WMSHistory to the MySQL backend of the Accounting system (which is in fact the default).
```

Comment:
```suggestion
This same agent can also report the WMSHistory to the MySQL backend used by the Accounting system (which is in fact the default).
```

Now that I see this: the current use case of running in LHCb production is to use ONLY the ES backend for WMSHistory. Is this a case considered here?

Reply:
So we want to monitoring WMSHistory _only_ on ES? Because in that case we should rethink this flag system as it takes Accounting as default in all cases.